Source/WebCore/platform/CaretAnimator.h

```diff
@@ -91,7 +96,9 @@ class CaretAnimator {
     PresentationProperties presentationProperties() const { return m_presentationProperties; }
     virtual void paint(const Node&, GraphicsContext&, const FloatRect&, const Color&, const LayoutPoint&) const;
     virtual LayoutRect repaintCaretRectForLocalRect(LayoutRect) const;
+#if defined(__has_include) && __has_include(<WebKitAdditions/RenderBlockAdditions.h>)
```

Comment:
Probably want a `FIXME` + bug, so we can remember to remove this `#if`?

Reply:
Good idea, I added a FIXME with a bug